### ISSUE TITLE
Question of prediction result

### ISSUE THREAD

**Issue by minekirito** [yashsmehta/personality-prediction#10]
Thank you your sharing! 

I run your project with my data successfully. But how can I see the prediction results of  "O" "C" "E" "A" "N" like TableⅡ in your paper. If it is convenient to you, please explain how to interpret the results generated by the codes in 'finetune_models'.

Thank you! Looking forward to your reply !!

**Comment by minekirito** [comment]
In addition, how to use models other than 'bert-base'！ Thank you!!

**Comment by Milad-Laly** [comment]
> In addition, how to use models other than 'bert-base'！ Thank you!!

I believe they are using LaTeX for the table as the screenshot seems to be from the paper. You could create a dataframe with the data and save it to a CSV.

**Comment by elhamadjidi** [comment]
@minekirito did you predict any new text using their pre-trained data?

**Comment by minekirito** [comment]
> @minekirito did you predict any new text using their pre-trained data?

No.  I think the performance of the model trained by this project is not enough for practical application. So I'm still working on how to train a better model.

**Comment by yashsmehta** [comment]
Hi @Milad-Laly @minekirito,
@edersoncorbari just added functionality to the repository to use our language model to make personality predictions on new text input! Please use:  `unseen_predictor.py`.
